capabilities-setup-config.json

```diff
@@ -0,0 +1,28 @@
+{
+    "pushNotifications": {
+        "android": {
+            "externalDependencies": [
+                "com.google.firebase:firebase-messaging:20.1.2",
+                "me.leolin:ShortcutBadger:1.1.22@aar"
+            ],
+            "packageListEntries": {
+                "imports": [
+                    "import io.invertase.firebase.messaging.RNFirebaseMessagingPackage;",
```

Comment:
necessary putting `import` here as there is really only one way to "import" files

Reply:
No that's not true in java you have static imports etc; rather than being too smart about it I prefer just acting dumb

Reply:
I even foresee people becoming creative ^^

Reply:
Fair enough